LayoutTests/ipc/request-digital-credentials.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<!-- webkit-test-runner [ IPCTestingAPIEnabled=true ] -->
```

Comment:
A little surprising to see this kind of test for other things than security testing?
Isn't there a way to leverage your new IPC code from JS without relying on window.IPC?

Reply:
Yes, hoping to get details on that. It's currently not documented.

Reply:
To be clear, right now not really (apart form very basic abort controller, maybe)... I'm trying to work my way to that by implementing the clients and a mock service.